doc/classes/ConfigFile.xml

```diff
@@ -100,8 +100,11 @@
 		</method>
 		<method name="encode_to_text" qualifiers="const">
 			<return type="String" />
+			<param index="0" name="full_objects" type="bool" default="false" />
 			<description>
-				Obtain the text version of this config file (the same text that would be written to a file).
+				Obtain the text version of this config file (the same text that would be written to a file). If [param full_objects] is [code]true[/code], encoding objects is allowed (and can potentially include code).
+				Internally, this uses the same encoding mechanism as the [method @GlobalScope.var_to_str] and [method @GlobalScope.var_to_str_with_objects] methods.
+				[b]Note:[/b] Not all properties are included. Only properties that are configured with the [constant PROPERTY_USAGE_STORAGE] flag set will be serialized. You can add a new usage flag to a property by overriding the [method Object._get_property_list] method in your class. You can also check how property usage is configured by calling [method Object._get_property_list]. See [enum PropertyUsageFlags] for the possible usage flags.
```

Comment:
See also godotengine/godot-proposals#7457.

Reply:
By the time this note was written `_validate_property` has been added which is better in these cases. Also may be worth mentioning `@export`.
```suggestion
				[b]Note:[/b] Not all properties are included. Only properties that are configured with the [constant PROPERTY_USAGE_STORAGE] flag set will be serialized. You can add an usage flag to a property by overriding the [method Object._validate_property] method in your class. In GDScript you can also use the [annotation @GDScript.@export] annotation. See [enum PropertyUsageFlags] for the possible usage flags.
```